### PR TITLE
support scala3

### PR DESCRIPTION
**Pull request by GijsbertvanVliet**

### CODE REVIEW

**Comment by regis-leray** [comment]
thank you. can you please update the github ci  to add a build for scala 3

**Comment by GijsbertvanVliet** [comment]
Aah I forgot. Sorry bout that. Added github ci

**Review by regis-leray — APPROVED** [review]

**Review by regis-leray — CHANGES_REQUESTED** [review]
please fix build

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -68,9 +68,9 @@ jobs:
       - name: Test 2.x
         if: ${{ !startsWith(matrix.scala, '3.') }}
         run: ./sbt ++${{ matrix.scala }} zio-ftp/test
-#      - name: Test 3.x
-#        if: ${{ startsWith(matrix.scala, '3.') }}
-#        run: ./sbt ++${{ matrix.scala }}! testDotty
+      - name: Test 3.x
+        if: ${{ startsWith(matrix.scala, '3.') }}
+        run: ./sbt ++${{ matrix.scala }}! testDotty
```
Comment (regis-leray) [inline .github/workflows/ci.yml]: use test

Comment (GijsbertvanVliet) [reply .github/workflows/ci.yml]: Hope It'll work now.
I don't think there's a way for me to check whether the command will work without you first triggering the github ci build is there?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -68,9 +68,9 @@ jobs:
       - name: Test 2.x
         if: ${{ !startsWith(matrix.scala, '3.') }}
         run: ./sbt ++${{ matrix.scala }} zio-ftp/test
-#      - name: Test 3.x
-#        if: ${{ startsWith(matrix.scala, '3.') }}
-#        run: ./sbt ++${{ matrix.scala }}! testDotty
+      - name: Test 3.x
+        if: ${{ startsWith(matrix.scala, '3.') }}
+        run: ./sbt ++${{ matrix.scala }} zio-ftp/test
```
Comment (regis-leray) [inline .github/workflows/ci.yml:69]: you could merge both steps by removing the condition

```
 - name: Test
    run: ./sbt ++${{ matrix.scala }} zio-ftp/test 
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (regis-leray) [reply .github/workflows/ci.yml]: you can test locally. :) 
i dont understand why i need to approve multiple times

Comment (regis-leray) [reply .github/workflows/ci.yml]: thank you again for your work

File: .github/workflows/ci.yml (hunk 2 above)
Comment (regis-leray) [reply .github/workflows/ci.yml:69]: and after that we will merge it

Comment (GijsbertvanVliet) [reply .github/workflows/ci.yml:69]: True, Done

File: .github/workflows/ci.yml (hunk 1 above)
Comment (GijsbertvanVliet) [reply .github/workflows/ci.yml]: Yes, I did test locally :)
I was just a bit unsure/unfamiliar with the notation in this ci file.

**Comment by GijsbertvanVliet** [comment]
Hi,
Could someone give an indication of when a release of this project with these changes will be created?
We would like to use zio-ftp in our scala3 project.
Thanks in advance!

**Comment by regis-leray** [comment]
@GijsbertvanVliet 
https://github.com/zio/zio-ftp/releases/tag/v0.4.1

**Pull request merged**